app/views/purchases/new.html.erb

```diff
@@ -0,0 +1,37 @@
+  <%= link_to group_purchases_path(params[:group_id]), data: { turbo: false }  do %>
+    <%= inline_svg_tag "chevron_left.svg", class: "chevron" %>
+  <% end %>
+  <h1>New transaction</h1>
+  <div id="whatever">
+    <%= inline_svg_tag "menu.svg", class: "burger-menu" %>
+  </div>
+</header>
+
+<main class="with_actions for_form">
+  <% flash.each do |name, msg| -%>
+    <%= content_tag :div, msg, class: name %>
+  <% end -%>
+  
+  <%= form_with(url: group_purchases_path, model: @purchase, class: "form") do |f| %>
+    <div class="field">
+      <%= f.label :name, "name" %>
+      <%= f.text_field :name, autofocus: true, autocomplete: "name", placeholder: "Bodyshop cream" %>
+    </div>
+
+    <div class="field">
+      <%= f.label :amount, "amount, $" %>
+      <%= f.text_field :amount, autofocus: true, autocomplete: "amount", placeholder: "15" %>
+    </div>
+
+    <div class="checkbox">
+      <%= f.label :amount, "categories", class: "categories_label" %><br />
+      <%= f.collection_check_boxes :group_ids, current_user.groups, :id, :name, :include_hidden => false %>
+      <span>No matching category?</span>
+      <%= link_to "add a new category", new_group_path, data: { turbo: false }, class: "static"  %>
+    </div>
+
+    <div id="actions">
+      <%= f.submit "add transaction", data: { turbo: false }, class: "btn"  %>
+    </div>
+  <% end %>
+</main>
```

Comment:
- [ ] When I am on the new transaction page there should be a back arrow button on the navbar, just like the one on the new category page that takes the user to the transactions page.

  ![image](https://github.com/katarighe/budget-app/assets/54848363/ba6e2a4b-9adf-4ab5-b742-22784f5f205b)

Reply:
✅ Fixed